mscore/preferences.cpp

```diff
@@ -995,15 +991,11 @@ void PreferenceDialog::updateValues()
       myPlugins->setText(prefs.myPluginsPath);
       mySoundfonts->setText(prefs.mySoundfontsPath);
 
-      idx = 0;
-      int n = sizeof(exportAudioSampleRates)/sizeof(*exportAudioSampleRates);
-      for (;idx < n; ++idx) {
-            if (exportAudioSampleRates[idx] == prefs.exportAudioSampleRate)
-                  break;
-            }
-      if (idx == n)     // if not found in table
-            idx = 0;
-      exportAudioSampleRate->setCurrentIndex(idx);
+      index = exportAudioSampleRate->findText(QString("%1").arg(prefs.exportAudioSampleRate));
```

Comment:
You need to make sure that the text is not translated, or better set Data and use findData